packages/bull-vault/src/ZenBullStrategy.sol

```diff
@@ -89,8 +89,7 @@ contract BullStrategy is ERC20, LeverageBull {
         address _eulerMarketsModule
     )
         ERC20("Bull Vault", "BullVault")
```

Comment:
This should be Zen Bull Vault and ZenBull imo.  I think this is the most important part to fix thb.